Angular/login/src/app/components/add-exchange/add-exchange.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Exchange } from 'src/app/models/exchange-model';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405403)
```suggestion
import { Exchange } from "src/app/models/exchange-model";
```